pkg/statser/statser_internal.go

```diff
@@ -34,7 +34,7 @@ type InternalStatser struct {
 
 // NewInternalStatser creates a new Statser which sends metrics to the
 // supplied InternalHandler.
-func NewInternalStatser(bufferSize int, tags gostatsd.Tags, namespace, hostname string, handler InternalHandler) Statser {
+func NewInternalStatser(bufferSize int, tags gostatsd.Tags, namespace, hostname string, handler InternalHandler) *InternalStatser {
```

Comment:
We can always make this return an interface that augments `Statser` and adds a `Run` method.